script/import_scripts/phpbb3/database/database_3_0.rb

```diff
@@ -13,7 +13,7 @@ def count_users
       SQL
     end
 
-    def fetch_users(last_user_id)
+    def fetch_users(last_user_id, _)
```

Comment:
Even though the argument is not used, I think we should still name it. The `_`  prefix is enough to indicate it's not being used?